docs/using-the-compiler.rst

```diff
@@ -341,7 +341,9 @@ Input Description
           "def": {
             "MyContract": [ "abi", "evm.bytecode.opcodes" ]
           }
-        }
+        },
+        // Choose which model checker engine to use: all (default), bmc, chc, none.
+        "modelCheckerEngine": "chc"
```

Comment:
Do we want to include the state of this setting in medatata (in `CompilerStack::createMetadata()`)? Or is it not necessary because SMT checker cannot affect compilation output?

Reply:
Yea I think it's not necessarily because it doesn't affect bytecode generation.

Reply:
OK. Fine.